src/hidpp20.c

```diff
@@ -2959,6 +2957,10 @@ hidpp20_onboard_profiles_write_profile(struct hidpp20_device *device,
 
 	memcpy(pdata->profile.name.txt, profile->name, sizeof(profile->name));
 
+	// This field needs to be zero for leds to work in G705
+	// If a non zero value is set, custom animation will be used instead
+	pdata->profile.free[0] = 0x00;
```

Comment:
I guess `free` implies it was assumed to be unused.
Could you then change the `free` member of the struct to only be one-byte wide, and add additional field like `set_to_zero_for_leds_for_g705`?
Be wary of endianness, the new field may need to go after the now-one-byte-wide 'free' field.

Reply:
I named it `custom_animation_g705` since it appears to control the custom animation.

And this field is definitely only one byte long since the other `free` byte is actually the first byte of the crc in this device, because it has a profile sector size of 255 bytes not 256